### DISCUSSION TITLE
tkintermapview

### DISCUSSION THREAD

**Discussion by SamMariina** [TomSchimansky/CustomTkinter#1845] [General]
Hello, Tom!

**Comment by SamMariina** [comment]
Ihave a problem with using tkintermapview: "No module named 'tkintermapview". Everything was installed good. I work in PyCharm. Is this mistake often in PyCharm?

**Reply by kosh66** [reply]
Did you pip install it?

**Reply by SamMariina** [reply]
yes, it was successfully installed. And earlier pip was updated

**Reply by SamMariina** [reply]
I just managed to run the code without errors))) I installed tkintermapview directly through the settings in Python Interpreter manually.
